inc/classes/class-google-auth.php

```diff
@@ -310,7 +310,7 @@ public function authenticate_user( $user = null ) {
 		$is_mu_site = is_multisite();
 
 		$token = Helper::filter_input( INPUT_GET, 'code', FILTER_SANITIZE_STRING );
-		$state = Helper::filter_input( INPUT_GET, 'state', FILTER_SANITIZE_STRING );
```

Comment:
@deepaklalwani97 

What is the diff between adding and not adding `Helper::` here?

Reply:
The encoded URL behaves differently in both fpm-fcgi and cli it gives null in cli that's why Helper method was added. So can we remove sanitize_text_field as it is already being sanitized by filter_input? as I have removed the Helper method here the test cases are failing.